iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ThreeWindingsTransformerImpl.java

```diff
@@ -139,10 +138,6 @@ public RatioTapChangerImpl getRatioTapChanger() {
             return ratioTapChanger;
         }
 
-        public Optional<RatioTapChanger> getOptionalRatioTapChanger() {
-            return Optional.ofNullable(ratioTapChanger);
```

Comment:
Not sure you should remove this implementation (even if this method has a default implementation with the same behavior), it makes it explicit that you are using the `ratioTapChanger` field